airqo-platform/AirQo-frontend
# website fixes

#### Summary of Changes (What does this PR do?)
- This PR fixes database connection error generated when the user attempts to access the website using www.airqo.net or www.airqo.africa. 
- Fix `NAN` and `Last Updated 50 year ago` error on website 
- Fix access to `https://airqo.org`
- Fix `wehubit` logo

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Just head over to production and try accessing the website using www.airqo.net or www.airqo.africa. You should not see any database connection error .

#### What are the relevant tickets?
- [ticket_id]()

#### Screenshots (optional)
